### PR TITLE
Feat(eos_cli_config_gen): router multicast ipv4 counters support

### LINKED ISSUE #1581: Feat(eos_cli_config_gen): router multicast ipv4 counters support
### Enhancement summary

Add router_multicast['ipv4']['counters'] to the model

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

This is a default configuration that gets populated / displayed in the running config (not hidden) when you activate router multicast for ipv4.

### Describe the solution you would like

```yaml
router_multicast:
  ipv4:
    counters:
      rate_period_decay: 300
```

```eos
router multicast
   ipv4
      counters rate period decay 300 seconds
```

### Describe alternatives you have considered

_No response_

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by ryanmerolle**
## Change Summary

This is a default configuration that gets populated / displayed in the running config (not hidden) when you activate router multicast for ipv4.

## Related Issue(s)

Fixes #1581

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes

```yaml
router_multicast:
  ipv4:
    counters:
      rate_period_decay: 300
```

```eos
router multicast
   ipv4
      counters rate period decay 300 seconds
```

### User Checklist

- N/A

### Repository Checklist

- [X] My code has been rebased from devel before I start
- [X] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [X] My change requires a change to the documentation and documentation have been updated accordingly.
- [X] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by ClausHolbechArista — COMMENTED** [review]
Please add molecule coverage for this feature

**Comment by ryanmerolle** [comment]
molecule test included

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md [hunk 1]
```diff
@@ -1899,6 +1899,8 @@ ip_igmp_snooping:
 ```yaml
 router_multicast:
   ipv4:
+    counters:
+      rate_period_decay: < 0-600 >
```
Comment (ankudinov) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md]: I'd mark it as optional in README.

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-multicast.cfg [hunk 2]
```diff
@@ -14,6 +14,7 @@ interface Management1
 !
 router multicast
    ipv4
+      counters rate period decay 300
```
Comment (ankudinov) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-multicast.cfg]: 4.27.0F (cEOS) asks for `counters rate period decay 300 seconds`.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md (hunk 1 above)
Comment (ankudinov) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md]: Please also specify that the interval is in seconds.

**Review by ankudinov — CHANGES_REQUESTED** [review]

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md (hunk 1 above)
Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md]: everything (ish) in eos_cli_config_gen is optional.

Comment (ryanmerolle) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md]: Yea nothing is mandatory.  I’m confused on the feedback given this fact.

I’m happy to specify the interval is in seconds, but I would need to change the key to something different than what eos calls it.  I do not think that is done anywhere in eos_cli_config_gen

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-multicast.cfg (hunk 2 above)
Comment (ryanmerolle) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-multicast.cfg:17]: I’ll double check.  I was using the latest M release for testing.

Comment (ryanmerolle) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-multicast.cfg]: This has been corrected.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md (hunk 1 above)
Comment (ankudinov) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md]: To be clear, my suggestion was: `rate_period_decay: < 0-600 >  # optional, seconds`
As we agreed that optional is not required, just: `rate_period_decay: < 0-600 >  # seconds`
Simply a comment providing additional info about the variable.

Comment (ankudinov) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md]: @ryanmerolle

Comment (ryanmerolle) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md]: Done.  Thanks

**Review by ankudinov — APPROVED** [review]
LGTM
changed the comment

**Comment by ryanmerolle** [comment]
> LGTM changed the comment

Thanks.  I forgot the in seconds comment since I was running around in the datacenter today while updating this from my phone.

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**